lib/transports-uws/polling.ts

```diff
@@ -122,6 +122,18 @@ export class Polling extends Transport {
       return;
     }
 
+    const contentLengthHeader = Number(req.headers["content-length"]);
```

Comment:
If the content is chunked (`Transfer-Encoding: chunked`), won't the `Content-Length` header be undefined there?

Reply:
Chunked POST's from the client are not supported by uWS, see https://github.com/uNetworking/uWebSockets.js/issues/669 the Socket.io client needs to not use it for this case

Reply:
OK, thanks for the link. But in that case, what happens in the issue? Does it throw on the first chunk?

Reply:
I see there is specific http code 411 for Length Required, I will update that https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/411

Reply:
Ok I tested this with new Node v17.5 `fetch` using `{ body: createReadStream() }` to send `transfer-encoding: chunked` request, uWS.js successfully responds with status `411 Length Required` when request is missing `content-length` header